corehq/apps/es/cases.py

```diff
@@ -21,6 +21,9 @@
 
 class CaseES(HQESQuery):
     index = 'cases'
+    default_filters = {
+        'is_commcare_case': filters.doc_type("CommCareCase"),
```

Comment:
I'm not sure if this is the "right" thing to do. Currently anything that uses `CaseES` is returning all cases including ones that have been deleted. That doesn't seem like what we should be doing, and it's inconsistent with what couch/sql do by default.